src/main.ts

```diff
@@ -224,6 +224,22 @@ function isSupportedPlatform(platform: PlatformInformation): boolean {
 }
 
 async function ensureRuntimeDependencies(extension: vscode.Extension<CSharpExtensionExports>, eventStream: EventStream, platformInfo: PlatformInformation, installDependencies: IInstallDependencies): Promise<boolean> {
+    await activateSdkExtension(eventStream);
     return installRuntimeDependencies(extension.packageJSON, extension.extensionPath, installDependencies, eventStream, platformInfo);
```

Comment:
Hmm, I believe we'll want to activate the SDK extension after all of the runtime dependencies have been installed but before they've been activated / have the chance to complain about a missing SDK 😄.

Also, since the SDK extension provides a path to dotnet the debugger piece will need to conditionally error because the current VSCode instance wont have the newly acquire dotnet sdk on the path yet since it will be in the midst of being acquired.

Reply:
> I believe we'll want to activate the SDK extension after all of the runtime dependencies have been installed but before they've been activated / have the chance to complain about a missing SDK

I don't know about that. We aleady acquire dependencies serially. I think it would be best to just treat the SDK as the most important dependency.